README.md

```diff
@@ -136,6 +133,52 @@ sendDistributionMetric(
 
 If your Lambda function is associated with a VPC, you need to ensure it has access to the [public internet](https://aws.amazon.com/premiumsupport/knowledge-center/internet-access-lambda-function/).
 
+## Trace & Log Correlation
+
+### Using the Datadog Tracer
+
+If you are using the [Datadog Tracer](#datadog-tracer-experimental), your log messages
```

Comment:
This isn't quite true. You still need to follow the steps (here)[https://docs.datadoghq.com/tracing/connect_logs_and_traces/nodejs/]

Reply:
👍 Thanks. Fixed.